app/components/base/form/form-page/form-page.component.ts

```diff
@@ -57,7 +57,7 @@ export class FormPageComponent {
     public openedWith: FocusableElement;
 
     // tslint:disable-next-line:no-forward-ref
-    constructor( @Inject(forwardRef(() => ComplexFormComponent)) private form: ComplexFormComponent) { }
```

Comment:
I think the formatter adds the space can you check it does that for you so we don't change that back on forth